### PR TITLE
feat: update node bindings

### PR DESCRIPTION
**Pull request by matthewkeil**
Updates README.md with new DAS functions.

**note: also turned on exceptions for node.  Is a bug that needed to be fixed.  Node can potentially throw errors and if exceptions are not turned on there are additional checks during allocations and getting data from the isolate that need to happen.  Those checks are extremely verbose.  Is better to turn on exceptions**

https://github.com/nodejs/node-addon-api/blob/main/doc/error_handling.md#handling-errors-with-maybe-type-and-c-exceptions-disabled

### CODE REVIEW

File: bindings/node.js/README.md [hunk 1]
```diff
@@ -7,7 +7,7 @@ API. The core functionality was originally a stripped-down copy of
 since then. This package wraps that native `c-kzg` C code in C/C++ NAPI
 bindings for use in node.js applications.
 
-Spec: https://github.com/ethereum/consensus-specs/blob/dev/specs/eip4844/polynomial-commitments.md
+Spec: <https://github.com/ethereum/consensus-specs/blob/dev/specs/eip4844/polynomial-commitments.md>
```
Comment (jtraglia) [inline bindings/node.js/README.md:10]: This link is broken. Can we replace it with the following two links:

* https://github.com/ethereum/consensus-specs/blob/dev/specs/deneb/polynomial-commitments.md
* https://github.com/ethereum/consensus-specs/blob/dev/specs/_features/eip7594/polynomial-commitments-sampling.md

**Review by jtraglia — COMMENTED** [review]
Looks good, thanks! Just one small thing.

File: bindings/node.js/README.md (hunk 1 above)
Comment (matthewkeil) [reply bindings/node.js/README.md]: Updated.

**Comment by matthewkeil** [comment]
@jtraglia Converted to draft until the other PR get approved/denied as I move the exception commit over and will revert that here

**Review by jtraglia — APPROVED** [review]

**Pull request merged**